onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc

```diff
@@ -2644,6 +2817,7 @@ static OpBuilderRegistrations CreateOpBuilderRegistrations() {
   }
 
   NNAPI_EP_ADD_SINGLE_OP_BUILDER("Elu", EluOpBuilder);
+  NNAPI_EP_ADD_SINGLE_OP_BUILDER("Slice", SliceOpBuilder);
```

Comment:
>NNAPI_EP_ADD_SINGLE_OP_BUILDER [](http://example.com/codeflow?start=2&length=30)

why are some of these `NNAPI_EP_ADD_SINGLE_OP_BUILDER`'s in separate blocks?

Reply:
There are just in order of when the op support was added.
Changed to sorted order

Reply:
I was wondering why we have
```
NNAPI_EP_ADD_SHARED_OP_BUILDER(...);
{
  NNAPI_EP_ADD_SHARED_OP_BUILDER(...);
}
NNAPI_EP_ADD_SHARED_OP_BUILDER(...);
```